include/charmlite/core/collection.hpp

```diff
@@ -379,6 +400,27 @@ namespace cmk {
             auto& idx = msg->dst_.endpoint().chare;
             this->buffers_[idx].emplace_back(std::move(msg));
         }
+
+        template<typename MapperType = Mapper<index_type>>
+        typename std::enable_if<
+        !(std::is_same<nodegroup_mapper<index_type>, MapperType>::value ||
+            std::is_same<group_mapper<index_type>, MapperType>::value)>::type
+        send_location_update_(chare_index_t idx, int home_pe, int loc)
```

Comment:
These can be moved into the `collection_bridge_`, since it's meant to encapsulate type-specific behaviors.